src/ripple/app/paths/impl/DirectStep.cpp

```diff
@@ -444,8 +459,7 @@ DirectStepI::validFwd (
 
     bool srcRedeems;
     IOUAmount maxSrcToDst;
-    std::tie (maxSrcToDst, srcRedeems) =
-            maxFlow (sb, src_, dst_, currency_);
+    std::tie (maxSrcToDst, srcRedeems) = maxFlow (sb, in.iou);
```

Comment:
Probably want `cache_->srcToDst` here, not `in.iou` (if we leave the "Ignore trust lines logic" inside `maxFlow`, which we need to think about)